kibana-reports/public/components/report_definitions/report_trigger/report_trigger_constants.tsx

```diff
@@ -31,7 +31,7 @@ export const SCHEDULE_REQUEST_TIME_OPTIONS = [
   },
   {
     id: 'cronBasedOption',
-    label: 'Cron based',
+    label: 'Cron-based',
```

Comment:
why is this changed? I assume this label field will be the value that client passes to server, which is defined as `Cron based` in schema

Reply:
This is only the label that the user will see, and a hyphen was requested in the UI final review.